src/deepsparse/yolov8/utils.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This code is fully ported from the ultralytics repository:
+https://github.com/ultralytics/ultralytics/blob/main/ultralytics/yolo/utils/ops.py
```

Comment:
Since we are requiring `ultralytics` in our dependencies anyway, do we need to port these functions?

Reply:
Good point. I will remove the superfluous code for now. We can later take further steps towards making this pipeline independent from `ultralytics` (e.g. by unifying "our" NMS function to serve both v8 and v5).